src/modules/0.0.8/aws_ec2/index.ts

```diff
@@ -11,13 +11,16 @@ export const AwsEc2Module: Module2 = new Module2({
   ...metadata,
   utils: {
     instanceMapper: async (instance: AWSInstance, ctx: Context) => {
+      const client = await ctx.getAwsClient() as AWS;
       const out = new Instance();
-      out.instanceId = instance.InstanceId;
+      out.instanceId = instance.InstanceId as string;
```

Comment:
Maybe instead of this assertion it early returns undefined if `InstanceId` somehow comes back null from AWS?

Reply:
I am almost certain this should never come back `undefined`, it is just the AWS type weirdness

Reply:
got it, that makes sense. I was mostly following the pattern in other places, but we should probably replace those cases with this patten